pkgs/top-level/all-packages.nix

```diff
@@ -19966,10 +19966,7 @@ with pkgs;
   # without atlas. Etc.
   liblapackWithAtlas = callPackage ../development/libraries/science/math/liblapack {};
   liblapackWithoutAtlas = liblapackWithAtlas.override { atlas = null; };
-  liblapack_3_5_0WithAtlas = callPackage ../development/libraries/science/math/liblapack/3.5.0.nix {};
-  liblapack_3_5_0WithoutAtlas = liblapack_3_5_0WithAtlas.override { atlas = null; };
   liblapack = liblapackWithAtlas;
-  liblapack_3_5_0 = liblapack_3_5_0WithAtlas;
```

Comment:
If we're removing these, the comment right above this needs to be updated as well. (since it mentions 3.5.0, etc.)

Reply:
Ah right somehow I never read that comment. It doesn't really explain *why* it was pinned at 3.4.1 either.

Should I just remove it? I doesn't really provide any value anymore.